src/components/backend-ai-agent-summary-list.ts

```diff
@@ -0,0 +1,501 @@
+/**
+ @license
+Copyright (c) 2015-2022 Lablup Inc. All rights reserved.
+*/
+
+import {get as _text, translate as _t} from 'lit-translate';
+import {css, CSSResultGroup, html, render} from 'lit';
+import {customElement, property, query} from 'lit/decorators.js';
+import {BackendAIPage} from './backend-ai-page';
+
+import '@vaadin/vaadin-grid/vaadin-grid';
+import '@vaadin/vaadin-grid/vaadin-grid-column';
+import '@vaadin/vaadin-grid/vaadin-grid-sort-column';
+import '../plastics/lablup-shields/lablup-shields';
+
+import '@material/mwc-linear-progress';
+import '@material/mwc-icon-button';
+import '@material/mwc-switch';
+import '@material/mwc-list';
+import '@material/mwc-list/mwc-list-item';
+import '@material/mwc-icon/mwc-icon';
+
+
+import {default as PainKiller} from './backend-ai-painkiller';
+import {BackendAiStyles} from './backend-ai-general-styles';
+import {IronFlex, IronFlexAlignment} from '../plastics/layout/iron-flex-layout-classes';
+import './backend-ai-list-status';
+import './backend-ai-dialog';
+import './lablup-progress-bar';
+
+/**
+Backend.AI Agent Summary List
+
+Example:
+
+<backend-ai-agent-summary-list active=true>
+... content ...
+</backend-ai-agent-summary-list>
+
+@group Backend.AI Web UI
+@element backend-ai-agent-list
+*/
+
+@customElement('backend-ai-agent-summary-list')
+export default class BackendAIAgent extends BackendAIPage {
+  @property({type: String}) condition = 'running';
+  @property({type: Boolean}) useHardwareMetadata = false;
+  @property({type: Array}) agents = [];
+  @property({type: Object}) list_status = Object();
+  @property({type: Object}) agentsObject = Object();
+  @property({type: Object}) agentDetail = Object();
+  @property({type: Object}) notification = Object();
+  @property({type: Object}) agentDetailDialog = Object();
+  @property({type: Object}) agentSettingDialog = Object();
+  @property({type: Boolean}) enableAgentSchedulable = false;
```

Comment:
And we don't need to expose `enableAgentSchedulable` as a property. let's just remove the property decorator and add access modifier.